tests/ltp/patches/fix_epoll_epoll-ltp.patch

```diff
@@ -0,0 +1,42 @@
+This change is necessary to remove the failure in test case
+“testcases/kernel/syscalls/epoll/epoll-ltp”. This 
+test case is failed because sgx-lkl-musl library API,
+“epoll_create” is successes unexpectedly, when negative
+value is passed to size parameter. 
+
+As per Musl library function implementation,
+size parameter is ignored. Test case is modified to print pass
+message when “epoll_create” returns valid fd for –ve size
+parameter value to run in sgx-lkl environment.
+
+diff --git a/testcases/kernel/syscalls/epoll/epoll-ltp.c b/testcases/kernel/syscalls/epoll/epoll-ltp.c
+index 12504ab95..e87eeda28 100644
+--- a/testcases/kernel/syscalls/epoll/epoll-ltp.c
++++ b/testcases/kernel/syscalls/epoll/epoll-ltp.c
+@@ -212,20 +212,16 @@ int test_epoll_create(unsigned int num_rand_attempts)
+ 	errno = 0;
+ 	fd_set_size = -1;
+ 	num_epoll_create_test_calls++;
++	// The Size parameter is ignored in musl library
+ 	epoll_fd = epoll_create(fd_set_size);
+ 	if (epoll_fd >= 0) {
+-		tst_resm(TFAIL | TERRNO,
+-			 "epoll_create with negative set size succeeded unexpectedly");
+-		num_epoll_create_test_fails++;
++		tst_resm(TPASS,
++			 "epoll_create with negative set size succeeded");
```

Comment:
Can you add to message that size parameter ignored by musl

Reply:
done